_studio/mfx_lib/encode_hw/h264/src/mfx_h264_encode_hw.cpp

```diff
@@ -733,6 +733,65 @@ void ImplementationAvc::DestroyDanglingCmResources()
         }
     }
 }
+/*Class for modifying and restoring VideoParams.
+  Needed for initialization some components */
+class ModifiedVideoParams
+{
+public:
+    ModifiedVideoParams() :
+        m_bInit(false),
+        m_RateControlMethod(0),
+        m_LookAheadDepth(0),
+        m_MaxKbps(0)
+    {}
+    void ModifyForDDI(MfxVideoParam & par, bool bSWBRC)
+    {
+        if (!bSWBRC)
+            return;
+        if (!m_bInit)
+            SaveParams(par);
+        // in the case of SWBRC driver works in CQP mode
+        par.mfx.RateControlMethod = MFX_RATECONTROL_CQP;
+    }
+    void ModifyForBRC(MfxVideoParam & par, bool bSWBRC)
+    {
+        mfxExtCodingOption2 & extOpt2 = GetExtBufferRef(par);
+        if (!(bSWBRC && extOpt2.MaxSliceSize))
+            return;
+        if (!m_bInit)
+            SaveParams(par);
+        // int the case of MaxSliceSize BRC works in VBR mode (instead LA)
```

Comment:
int -> in?

Reply:
@lakulako : thank you for review. Mind, please, that these are cherry-picks of the changes from master branch to release branch. While review is very much appreciated, these commits by itself **will not be changed**. Instead other changes which fix issues in these commits will  be appended. Thus, please, file bugs if you spot bugs to be fixed in master and/or PRs to address minor issues right away like here (I don't think that comment fix worth cherry-picking).